docs/release_notes/0.91.0.md

```diff
@@ -0,0 +1,18 @@
+# Release 0.91.0
+
+## Features
+
+- Removed legacy path for al2 and ubuntu (#4968)
+
+## Improvements
+
+- Migrate usage of SSM to aws-sdk-go-v2 (#5009)
+- Migrate ELB and ELBv2 to aws-sdk-go-v2 (#4937)
```

Comment:
```suggestion
```

I would remove these, since they aren't using facing (why would a use care about these changes? we don't include dependecy bumps like [this](https://github.com/weaveworks/eksctl/pull/5028) in the release notes). But I know we might differ on this so I'm going to approve regardless of if this stays or not :smile:

Reply:
I agree though. These are technical depts that we are working on. I don't think they have a place in a user facing release manifest.

Reply:
@Skarlso is this blocking you from approving the release notes?

Reply:
Not in particular, no. But I think in the future these notes can be omitted. :)

Reply:
> @Skarlso is this blocking you from approving the release notes?

Can we have a discussion about this though? I'm curious why you want to keep them in @Himangini